tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature

```diff
@@ -201,15 +208,16 @@ Feature: sharing
     But as "Alice" file "/folderToShare/fileInside" should not exist
 
 
-  Scenario: receiver tries to rename a received share with read permissions inside the Shares folder
```

Comment:
read permission?

Reply:
It was already written by someone else. Will not it be better to keep it like:
`receiver tries to rename a received share inside the Shares folder`

Reply:
> read permission?

"read permissions" means except "Can edit" permission